cli/args/flags.rs

```diff
@@ -3299,6 +3306,7 @@ fn coverage_parse(flags: &mut Flags, matches: &mut ArgMatches) {
     None => vec![],
   };
   let lcov = matches.get_flag("lcov");
+  let html = matches.get_flag("html");
```

Comment:
What happens if both `--lcov` and `--html` are specified? Maybe we should turn it into an enum to make this mutually exclusive?

Reply:
Added CoverageType enum